nano/templates/pda_chatroom.tmpl

```diff
@@ -0,0 +1,84 @@
+<!--
+Title: PDA Chatroom UI
+Used In File(s): \code\game\objects\items\devices\PDA\chatroom.dm
+ -->
+<div id="chatroom-header" class="item">
+	<div class="itemLabelNarrow">
+		<b>Chat Functions</b>:
+	</div>
+	<div class ="itemContent">
+		{{:helper.link(data.silent==1 ? 'Ringer: Off' : 'Ringer: On', data.silent==1 ? 'volume-off' : 'volume-up', {'choice' : "Toggle Ringer"}, null, 'pdalink fixedLeftWide')}}
+		{{:helper.link(data.toff==1 ? 'Notifications: Off' : 'Notifications: On',data.toff==1 ? 'close' : 'check', {'choice' : "Toggle Chatroom"}, null, 'pdalink fixedLeftWide')}}
+		{{:helper.link('Set Ringtone', 'bell-o', {'choice' : "Ringtone"}, null, 'pdalink fixedLeftWide')}}
+		{{:helper.link('New Room', 'plus', {'choice' : "New Room"}, null, 'pdalink fixedLeftWide')}}
+	</div>
+</div>
+
+<div id="chatroom-body">
+	{{if data.no_server}}
+		<h3 class="bad">ERROR: Messaging server is not responding.</h3>
+	{{else data.room}}
+		{{if data.inviting}}
+			<h3>Invite whom to #{{:data.room}}?</h3>
+			{{for data.people}}
+				{{:helper.link(value.name, 'user-plus', {'choice': "Invite PDA", 'user': value.ref}, null, 'pdalink fixedLeftWidest')}}
+			{{empty}}
+				<b>No other people found.</b>
+			{{/for}}
+		{{else}}
+			<h3>#{{:data.room}}&nbsp;&ndash;&nbsp;<span class="average">{{:data.topic}}</span></h3>
+			<div class="item">
+				<div class="itemLabelNarrow">
+					<b>Room Functions</b>:
+				</div>
+				<div class="itemContent">
+					{{:helper.link('Invite', 'user-plus', {'choice' : "Invite"}, null, 'pdalink fixedLeftWide')}}
+					{{:helper.link('Change Topic', 'tag', {'choice' : "Topic"}, null, 'pdalink fixedLeftWide')}}
+					{{:helper.link('Leave Channel', 'user-times', {'choice' : "Leave"}, null, 'pdalink fixedLeftWide')}}
+					{{:helper.link(data.auto_scroll ? 'Autoscroll: On' : 'Autoscroll: Off', 'level-down', {'choice' : "Autoscroll"}, null, 'pdalink fixedLeftWide')}}
+				</div>
+			</div>
+			<div style="width: 75%; float: right; margin-left: 1em">
+				<div id="historyView" class="statusDisplay">
+					{{var prevName = "";}}
+					{{for data.history}}
+						<div class="item">
+							<div class="itemLabelNarrow">{{:value.username!=prevName ? value.username : "&nbsp;"}}</div>
+							<div class="itemContent">
+									{{:value.message}}
+							</div>
+						</div>
+						{{prevName = value.username;}}
+					{{/for}}
+				</div>
+				<div>
+					{{:helper.link('Post', 'comment', {'choice' : "Post"}, null, 'pdalink fixedLeftWidest')}}
+				</div>
+			</div>
+			<div id="userView" class="statusDisplay">
+				{{for data.users}}
+					<div class="item">
+						<div class="itemLabel" style="width: 100%">{{:value}}</div>
+					</div>
+				{{/for}}
+			</div>
+			<script>
+				$(function() {
```

Comment:
`$(function)` is shorthand for `$(document).ready(function)` which has no direct basic javascript equivalent.